test/integration/controllers/v1/UserController.test.js

```diff
@@ -98,17 +98,20 @@ describe('POST /user/password', () => {
   });
 
   /* Only user_id field posted. */
-  it('should accept test `user_id` and return expected json payload', (done) => {
+  it('should prepare the payload for `user_id`', function (done) {
+    this.timeout(5000);
```

Comment:
I was thinking about it, but then it also might make sense to make it adjustable based on test, as they can take different time.